test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +11,19 @@
 
 public class UtilsTest {
 
+    @Test //(expected = NullPointerException.class)
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.